demo/nested.html

```diff
@@ -12,6 +12,8 @@
     /* make nested grids have slightly darker bg */
     .grid-stack.grid-stack-nested {
       background: #e4e4c1;
+      border: 2px dashed rgba(0, 0, 0, 0.4);
```

Comment:
are those CSS changes necessary ? I though the blended background was enough to distinguish the nesting - adding border will mess the padding and create unnecessary  scroll content/sizing issues...

I didn't pull in your changes to check.